resources/views/admin/settings/emails/test.blade.php

```diff
@@ -0,0 +1,40 @@
+@extends('admin.layouts.master')
+
+@section('main_content')
+    <div class="container-fluid">
+        <div class="row">
+
+            @include('admin.layouts.partials.subnav-settings')
+
+            <div class="col-sm-9 col-sm-offset-3 col-md-10 col-md-offset-2 main">
+
+                {!! Form::open(['action' => 'EmailController@sendTestEmail']) !!}
+
+                    <h1 class="page-header">Send Test Email</h1>
+
+                    <div class="panel-body">
+                        <div>
+                        {!! Form::label('recipient', 'To: ') !!}
+                        <br>
```

Comment:
these `<br>` seem really hacky to me but I couldn't figure out how to get it to look nicely like how it does in the partial!

Reply:
@chloealee what if you remove `class="panel-body"` (I think that's in the partial for the collapsing feature) or change the class to be `wrapper` or maybe `form-group`?

Reply:
If fiddling with it a bit doesn't seem to change anything then this is totally fine!!

Reply:
@katiecrane those didn't work either :-/ CSS and I do not get along! haha

Reply:
@chloealee no sweat, definitely not something worth spending any more time on and this is totally fine!